### PR TITLE
Add audit api doc

### PR DESCRIPTION
**Pull request by Sean-Holcomb**
Docs for the Audit Api

### CODE REVIEW

File: audit-api.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Audit API
+The Audit API apis a series of tests to the various cached data sources of the Kubecost to check the validity of the data sources and the processes that act upon then. Each of the Audits tests represent an invarience that should remain true dispite any changes that occur to the system as a whole. Each Audit is stored in an AuditSet, which besides audits contains a window for the timeframe that the Audits it contains cover. The Audits themselves each have a timestamp for their last run, a status, a discription and other audit specific structures which contain the results of the run.
+
+### Audit API
+#### GET `/etl/audit`
+Returns AuditSets for given window saved in the AuditStore
+
+Parameters:
+`window`: excepts all standard Kubecost window formats. excluding this argument returns all audits in range
```
Comment (nealormsbee) [inline audit-api.md:9]: `excepts` -> `accepts`

File: audit-api.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+#### Base Return types
+
+The base return types are used by each Audit result to express any descrepancys that have been detected.
+
+##### Audit Missing Value
+Audit Missing Value denotes a value that was present in one data source and not another. The presence of this base result will generally result in a Warning status in the Audit that contains it. It contains a `Description` of the missing value and the `key` which acts as a way of identifying the missing value.
+
+##### Audit Float Result
+Audit Float result represents a difference in float values between two data sources. The presence of this base result generally indicates a Failed status on the Audit. It contains an `Actual` value derived from the data source being audited and `Expected` value which is generated by the Audit itself.
```
Comment (nealormsbee) [inline audit-api.md:37]: capitalize Result ?

File: audit-api.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+Allocation Reconciliation Audit records the differences of between compute resources (cpu, ram, gpu) costs between
+allocations by nodes and node assets. It contains the additional fields:
+
+* `Resources`: A `map[string]map[string]*AuditFloatResult` records difference in resource values between asset and allocations. keyed on node name and resource name
+
+##### Total Audit
+Total Audit records the differences between a total store and the totaled results of the store. It contains the additional fields:
+
+* `TotalByNode`: A `map[string]*AuditFloatResult` recordes differences of calculated total by node. keyed on node name
+* `TotalByCluster`: A `map[string]*AuditFloatResult` recordes differences of calculated total by cluster. keyed on cluster id
```
Comment (nealormsbee) [inline audit-api.md:58]: `recordes` -> `records`

File: audit-api.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+##### Agg Audit
+Agg Audit contains the results of an Audit on an AggStore which checks if the aggregate of the store it draws from matches it.
+
+* `Results`: A `map[string]map[string]*AuditFloatResult`  records differences of calculate agg. keyed on aggregation prop and allocation or asset key
+
+##### Asset Reconciliation Audit
+Asset Reconciliation Audit records differences in assets and the Cloud items that it is able to successful match.
+
+* `Results`: A `map[string]map[string]*AuditFloatResult` Differences in cloud assets and matched assets. keyed on providerId and category
```
Comment (nealormsbee) [inline audit-api.md:68]: Capitalize `keyed` ?

**Review by nealormsbee — COMMENTED** [review]
YAAAAAS thank you.

Noted a few typos but otherwise looks great

**Comment by Sean-Holcomb** [comment]
Oof, I ran it through a spell checker, but I guess it didn't paste over properly. A few here I missed though, so thank you.

**Review by nealormsbee — APPROVED** [review]

**Pull request merged**